webpack.config.views.js

```diff
@@ -42,17 +42,7 @@ module.exports = (env, { mode }) => {
                             },
                             isModule: true,
                             sourceMaps: isDev,
-                            minify: !isDev,
                             jsc: {
-                                minify: {
-                                    compress: !isDev,
-                                    mangle: isDev
-                                        ? false
-                                        : {
-                                              keep_classnames: true,
-                                              keep_fnames: true,
-                                          },
-                                },
```

Comment:
are we indeed disabling minification, or is it happening by default anyway?

Reply:
It happens by default on another step. These flags don't make any sense here.

Reply:
Thank you for clarifying! This helps a lot.